### ISSUE TITLE
Gtk warnings

### ISSUE THREAD

**Issue by sparshg** [sugarlabs/readetexts#24]
Found while updating [flatpak version](https://github.com/flathub/org.sugarlabs.ReadETexts/pull/7#event-9954064305), also happens in when run natively with `sugar-activity3`

```
(sugarapp:2): Gtk-WARNING **: 04:49:02.700: Failed to set text '<span foreground='black' size='14000'>' from markup due to error parsing markup: Error on line 1 char 55: Element “markup” was closed, but the currently open element is “span”
no font size found
All done!
/app/share/sugar/activities/ReadETexts.activity/ReadEtextsActivity.py:554: Warning: g_value_get_int: assertion 'G_VALUE_HOLDS_INT (value)' failed
  self.num_page_entry.props.text = str(current_page + 1)
```

**Comment by quozl** [comment]
Yes, this first warning is caused by calling Gtk.Label.set_markup with unbalanced tags.  Also, the code calls set_markup then calls set_text, in two places, when it should only call set_markup.  GTK has changed since this code was written.  See https://lazka.github.io/pgi-docs/#Gtk-3.0/classes/Label.html#Gtk.Label.set_markup for current semantics.

The cause second warning isn't clear to me, it's just a Gtk.Entry, but some experimentation with the affected line may help.
